lib/themes/dosomething/paraneue_dosomething/template.php

```diff
@@ -92,6 +94,7 @@ function paraneue_dosomething_js_alter(&$js) {
 
   // Force removal of files that we can't seem to kill with exclude[js] array
   unset($js['profiles/dosomething/modules/contrib/devel/devel_krumo_path.js']);
+  unset($js['profiles/dosomething/modules/contrib/google_analytics/googleanalytics.js']);
```

Comment:
Ah ok. So its the .module that adds the snippet? http://cgit.drupalcode.org/google_analytics/tree/google_analytics.module?id=9bd941b#n328